lib/models/project.js

```diff
@@ -126,28 +126,7 @@ class Project {
     }
 
     this._bowerDirectory = this._bowerDirectory || 'bower_components';
-    logger.info('bowerDirectory: %s', this._bowerDirectory);
-  }
-
-  get bowerDirectory() {
-    deprecate(
-      [
-        '`bowerDirectory` has been deprecated.',
-        "If you still need access to the project's Bower directory, you will have to manually resolve the project's `.bowerrc` file, and read the `directory` property instead.",
-      ].join('\n'),
-      false,
-      {
-        for: 'ember-cli',
-        id: 'ember-cli.project.bower-directory',
-        since: {
-          available: '4.3.0',
-          enabled: '4.3.0',
-        },
-        until: '5.0.0',
-      }
-    );
-
-    return this._bowerDirectory;
+    logger.info('_bowerDirectory: %s', this._bowerDirectory);
```

Comment:
This PR just cleans up the deprecation. `_bowerDirectory` will be cleaned up separately.